pkgs/racket-doc/scribblings/foreign/types.scrbl

```diff
@@ -1736,7 +1736,19 @@ is to throw an exception.
     (_enum '(ok = 0
              invalid_input
              buffer_too_small)))
-]}
+]
+
+Note that the default basetype is @racket[_fixint]. This
+differs from C enumerations that can use any value in
+@racket[_ufixint]. Any @racket[_enum] using negative values
+should use @racket[_ufixint] for the base type.
```

Comment:
This seems backwards: shouldn’t this say that the default base type is `_ufixint`, and users who wish to use negative values need to use `_fixint`?

Reply:
Woops, yes, you are 100% correct.